manifests/apache.pp

```diff
@@ -123,6 +123,7 @@
         ssl_chain         => $pulpcore::apache_https_chain,
         ssl_ca            => $pulpcore::apache_https_ca,
         ssl_verify_client => $ssl_verify_client,
+        ssl_options       => '+StdEnvVars +ExportCertData',
```

Comment:
```suggestion
        ssl_options       => ['+StdEnvVars', '+ExportCertData'],
```

Reading https://github.com/puppetlabs/puppetlabs-apache/blob/e14ec095534469f06b417041d7d6967f232eabc5/manifests/vhost.pp#L1640-L1657 , I think this is the correct way to handle multiple options